lib/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module DangerSwiftlint
   VERSION = '0.30.2'
-  SWIFTLINT_VERSION = '0.46.2'
+  SWIFTLINT_VERSION = '0.50.0'
   SWIFTLINT_HASH = '20dbcc13788993dd27b1076a13ca647d'
```

Comment:
The hash should also be updated, see issue #178  and PR #179.

Reply:
Hey @mvandervelden, thx for the quick feedback. Maybe you could provide me the steps to create the SWIFTLINT_HASH correctly, did: md5 portable_swiftlint.zip

Reply:
That should be correct yes. I just double checked on the version that I did.